ext/droption/droption.h

```diff
@@ -337,7 +340,19 @@ template <typename T> class droption_t : public droption_parser_t
     droption_t(unsigned int scope_, std::string name_, unsigned int flags_,
                T defval_, std::string desc_short_, std::string desc_long_)
         : droption_parser_t(scope_, name_, desc_short_, desc_long_, flags_),
-        value(defval_), defval(defval_), has_range(false) {}
+        value(defval_), defval(defval_), valsep(DROPTION_DEFAULT_VALUE_SEP),
+        has_range(false) {}
+
+    /**
+     * Declares a new option of type T with the given scope, behavior flags,
+     * accumulated value separator, default value, and description in short and long
```

Comment:
s/separator/separator (see #DROPTION_FLAG_ACCUMULATE)/

(The # makes a doxygen link)